packages/TelegramClient-Core.package/TCCUser.class/properties.json

```diff
@@ -0,0 +1,17 @@
+{
+	"category" : "TelegramClient-Core",
+	"classinstvars" : [
+		 ],
+	"classvars" : [
+		 ],
+	"commentStamp" : "pk 5/9/2021 16:30",
+	"instvars" : [
+		"id",
+		"firstname",
+		"lastname",
+		"username" ],
```

Comment:
Please rename to camel case (`firstName`, `lastName`). We think leaving `username` would be fair since it is verbatim from the tdlib event.